src/gen_cranberry/gc_dev_xbee.cpp

```diff
@@ -4,6 +4,8 @@ void gc_dev_xbee_open(void)
 {
     soft_serial.begin(9600);
     xbee.begin(soft_serial);
+    digitalWrite(3, HIGH);
```

Comment:
Since this line of code is very abstract, you should include a comment describing what this does.